index.html

```diff
@@ -52,5 +58,220 @@ <h2 class="typography supportText">I’am a Software Developer</h2>
HYrdJC+73hBpZk4GRTwc4Z6BglnCCGVA6KPne3gStkTSOmqMwHP3CM9YMTDT/8hAiKcpVTIoHCOty6qpaiCViSxJuRJwNF/PvnZ0nhot1GWUCK2b6tyUGAM6KcFQlSDCU9WH7EyFVOG+mzAMs3CjNtiL80RQ8tMkPGwv4dODRIjCgbTKorsPkhInUWha74BSQJGiNFdxp4hqrQNERIvUIQ+RW63hIgY3MxVpKEVEqYnFCQAV8TTNd5DiEVQyOHLjDNTy5KvtJ/rrtEa/qVuw64Zh7T9b9ucS7sFYo2/e+nXjZZVEKfQFODXHSa1k8/Fzn1ocPTrb2Lxw5uGtU3eaGT3QNAL3qBRBUANhjrWeUMpIEB1HqEz95MjTDMGIw2dMqkDWg8aIZwUiEWVJPywgr20DND0AY/TJCs9gCF4hz7wJxYPubgdm3/7s5g9ecZKWUg4IwHJ3d19z9qa6tZY+5KNnH33kzCOJxv76b7z9Pe/5zdI058+fa9t2mhknAGuqW3TddO1uvFzvkL5rzR27MYc5KS1FvfbhU0rdQU1+865buNcpB/CWHeYOwTc78hAYYXv+MaMxOVIr5yEOG33QhQnimHnyoDKoj3SkPzoI91i7uXuMzWwkiUkECWUh41KzEyYggpChVpF0tgLhKgqEpZJQxlRC5CtSnmie8WEj9wNQphuPeXUtklQyAhQNd6ajF5Dvl9SN9MASEiwQiTAgRqZVJIdDSKdItjMiY58eBCHunswwMwtUh1CLqIgIBCigECqMohKCEFUzM3PXYrWaGUXCquVA7UY1lcZEWEVEQ9aSqCCSMkbSc+uAYHg4m6brRrHVBFAdmikIXuH12r3d40Fi+wzdcdnE/Upb98dgWOs/pHWVrzGvAGCrFxzTE6eO/NYn33dpfgzRkeo2aFPCxCiIKmiQORIYzeUoEm5e8lzWMTshApSMHaSMjuzhImQwuHbF2xMPAhCMCZ2xhsIiSaDrl1AkaiSkD6sRzn241dVUrBuchxURB7c4KTcMLykyi/CAVITu1aydAKxbt5M9fvz4m974li96/ReJ6P/7v/8b3Wx29OjRxWIxzZwTgDXVVE8JObp5BYl7W/50x7sDXP2Qs2OntZktzj9Ul9vTiXdwGNZsNlssl3arYlif3T/nlq66uLVH3fwzsXZDH4cjImoamDNGG6qgw0d+E5KDBHEJAeDwcIgwaRmkluKWHvDi7iJphxVcMybGkyos3dHTXzmEyhLuoAR8GKzKoKCICAXQVAQ5RoUhSXejB0QQRsr6TA2VQpi5FVlzKTKjEKGi7m7mRSUIWhq/M9xUJcbMQ7OaUJEjhYRMT3qhVjgQCmQ0Y2SjHME8AubBOniFaGMS6iECKY1QRBWqinBzq25Waq0yDBFN1Bo+uAcRUYegpxt9UAJK0dECP8WJa9eeIOlhHu3WUXDthk8+0VQ0KQqneipDfjzxQ9yHYWFNxcL+dEI8Rk6INYC9DmdgDLtH+dArnnfiP777t7vbXgqKmVFUxd1MSITHOiuVIhDhKKZNglVQZJ1DCoom9TKDSklCIFQK09mdoxgZBD2cFJAZqEoKlQDTyh2MWCNbwrzgbw6g6olqGIYDWleLiN5ImA7JojockIpwGJqmmQbKW6/M7Dl3Pef/9Zf/6utf90UkH3jwgVrrf/JNf/TYsWPf8c++fWd7e5ofJwBrqkOEQ11dqro31Zm7md2kWBXJtml8n/MRyaYUkLlQuP5Usif7jtwAPOshx7d8icjGfL5YLuuBcfWfdQTrMC9wDkG/OoZ7IQAmY8HHiD8EJMSBGMlKIuEOc/eIzPIyNzcTEZIemY6HcBsFgw7ZuzFOREgg4wpDBMgEvTTGYsADkkQIAgU0NzpczAXQUtgIUXSds8dMCwsw3N2ZOWIIMtwtKRXmLpS1ElE8HAgV8fAICMU4WtDHHn2Eyb2yalCVQKwBMJAUqCcnbMT+4EGRBLEY4SERbm4O1HDzOmhRuquoRhFAmiIsquFWShmqqLkbq7mKOWDuDgfgZgEY6HRP+pm4AhbMJEQGaU5oU7rO06N63+zDy4jkBFTdskuxg1ARPj6GhcenYuEqReEVMBb3/SFxpugXR4u/8dV3f/xSu7hwbn78hFsQoWIRxWN0VgclSDLGTMFYkx6Fo9wvP4vg6GeVJCoKJeCpMMb6ZQT2CFiM0QeLaXjHNc1qLVEUsCDgN3cM2cGtSZpSbrQhpWmaAwKwzCw8GbhT3Vp4Ryl//I/9ide8+jX/9J9/u6p+7df8wbZt+375VV/5jR/76Eff+m++T6cGagKwpjqclffbP/LxT/zWBz4wDLVpmhfc/dxX3nefiPgNvzK4KjSa586d//e//Csnjx970+tflw3bYrl826+9faj1DV/wBSeOH8udelow1kHd2YtwqwAo0xV9Pc7z+Wy2XC6HWxDDkjjMre+wcyt2vFedwIlVBeiEwtMvZkS0Mv0uI/UizdhVRSLtm8LrmO03Djhr6Gt8C0kELB9xJEcrkOQJBIQaGI3jyTH5L02sQhSiPvQOmNtQl8W0aVy1JMMCAKlgRHjCR3ALihCSfCnAfc3mkJC11w1IFU0SU8YHJreKBNxBoZChEeYBupuHilByRyhS4E4ygkEfNyQhLLG0ASMl3M2qkxauZqGNmUutpbbSFBFVVSkiqjZUU61VrTrcqA6PcAfSf97p4clfy2NFhghCxkjFdgMqTwJGTHXL1jWVEH4WDAtXUrHwdBSFjMdiWNYfxdl7b5v95sd/u7Sv1M3GxQVjkgSFnjrBnIOgaZOXLNExBzQ/wU1Ygglnp2/VKBJOFzyQwYCklDjCQ0RGCaJ7os4haQEvAacAIjAjws1u3v611uoHc6s4DYNuhPzBx6xURA6usxiGoeu6aby5tZZCcfr07W95y5f8wA++9Tu/61/+/t/3NXmL6ju/6zvvv//lX/IlX/pTP/OT2xMJawKwprrFLvun8jQVOXP23He99Qfe/s53nzl7NseAI0eOvPrl
```

Comment:
- [ ] Since the About section is a whole section itself, please wrap it in a `section` element.